CMakeLists.txt

```diff
@@ -35,10 +37,28 @@ target_link_libraries(playclouds
   yaml-cpp
 )
 
-add_executable(standalone
-  src/standalone.cpp
```

Comment:
The standalone application should still be build, or is there a reason to remove it?